user/forms.py

```diff
@@ -0,0 +1,27 @@
+import datetime
+
+from django import forms
+from django.contrib.auth.forms import UserCreationForm
+from django.forms import ModelForm
```

Comment:
以下のようにして、formsを使わなければ3行目は必要ないです
```suggestion
from django.forms import ModelForm,  SelectDateWidget
```